lib/pengine/complex.c

```diff
@@ -570,6 +570,22 @@ unpack_requires(pe_resource_t *rsc, const char *value, bool is_default)
                  (is_default? " (default)" : ""));
 }
 
+/*!
+ * \internal
+ * \brief Parse configuration XML for a given resource
+ *
+ * Unpack the XML object containing a resource's configuration into an empty
+ * \c pe_resource_t object.
+ *
+ * \param[in]  xml_obj   XML node containing the resource's configuration
+ * \param[out] rsc       Destination for unpacking
+ * \param[...] parent    Resource's parent (\c NULL if the resource is a
+ *                       primitive)
```

Comment:
in is sufficient here. The pointer can't be const since we're assigning it to a non-const struct member, but this function can't change it.

Reply:
That's just a typo -- I made placeholders and forgot to fill that one in.